pocs/camera/__init__.py

```diff
@@ -64,6 +65,8 @@ def create_cameras_from_config(config=None, logger=None, **kwargs):
     if not logger:
         logger = logger_module.get_root_logger()
 
+    logger.info(f'kwargs: {kwargs!r}')
```

Comment:
Maybe just a `debug` level on this.

Reply:
Dropped that, I should have removed it before creating the PR.